src/d/actor/d_a_npc_bs1.cpp

```diff
@@ -12,20 +12,22 @@
 #include "d/d_kankyo.h"
 #include "d/d_shop.h"
 #include "d/d_snap.h"
+#include "d/d_letter.h"
 #include "d/d_procname.h"
 #include "dolphin/types.h"
 #include "f_op/f_op_actor.h"
 #include "m_Do/m_Do_ext.h"
 #include "m_Do/m_Do_hostIO.h"
 #include "m_Do/m_Do_mtx.h"
+#include "m_Do/m_Do_controller_pad.h"
 
 // Needed for the .data section to match.
 static Vec dummy_2100 = {1.0f, 1.0f, 1.0f};
 static Vec dummy_2080 = {1.0f, 1.0f, 1.0f};
 static u8 dummy_1811[] = {0x02, 0x00, 0x02, 0x01};
 static f64 dummy4[2] = {3.0, 0.5};
 
-static uint l_msgId;
+static u32 l_msgId;
```

Comment:
```suggestion
static uint l_msgId;
```

FYI "msgId" is a process ID and should be uint, while "msgNo" is a string number in the BMG file and should be u32, they're separate concepts.

it looks like some of the member names we currently have in `msg_class` are wrong and get those two things mixed up and I assume that's where the confusion comes from, I'll fix those up later